### PR TITLE
[CLEANUP Beta] remove Object.defineProperty polyfil

### PR DESCRIPTION
**Pull request by stefanpenner**
(╯°□°）╯︵ ʃıɟʎʃod¬ ʎʇɹǝdoɹdǝuıɟǝp¬

### CODE REVIEW

File: packages/ember-metal/lib/main.js [hunk 1]
```diff
@@ -231,8 +228,8 @@ Ember.GUID_KEY        = GUID_KEY;
 Ember.create          = create;
 Ember.keys            = keys;
 Ember.platform        = {
-  defineProperty: defineProperty,
-  hasPropertyAccessors: hasPropertyAccessors
+  defineProperty: true, defineProperty,
```
Comment (thejameskyle) [inline packages/ember-metal/lib/main.js]: ^ `defineProperty: true,`~~`defineProperty,`~~

File: packages/ember-metal/lib/properties.js [hunk 2]
```diff
@@ -144,7 +139,7 @@ export function defineProperty(obj, keyName, desc, data, meta) {
       value = desc;
 
       // compatibility with ES5
-      objectDefineProperty(obj, keyName, desc);
+      Object.oefineProperty(obj, keyName, desc);
```
Comment (marcioj) [inline packages/ember-metal/lib/properties.js]: oefineProperty => defineProperty

Comment (rwjblue) [inline packages/ember-metal/lib/properties.js]: @marcioj - Great catch! Could you submit a PR fixing?

Comment (marcioj) [inline packages/ember-metal/lib/properties.js]: Sure. PR coming :smile:

**Pull request merged**